### ISSUE TITLE
only one contact matrix gets processed

### ISSUE THREAD

**Issue by jazberna1** [koszullab/chromosight#39]
Hello Cyril,

I have run chromosight 1.4.0. It did finishe but it seems to me it only processed one matrix, the EBV contact matrix that corresponds to Human_herpesvirus_4 in the GRCh38 fasta reference file. Is there something I am missing to set? 

This the log:
_pearson set to 0.3 based on config file.
min_separation set to 5000 based on config file.
max_perc_undetected set to 50.0 based on config file.
max_perc_zero set to 10.0 based on config file.
Matrix already balanced, reusing weights
Preprocessing sub-matrices...
 [====================] 100.0% EBV-EBV
Detecting patterns...
 [--------------------] 0.0% Kernel: 0, Iteration: 0
 [--------------------] 0.0% Kernel: 0, Iteration: 0
No pattern detected ! Exiting._

This is how I run the program:
```
/mnt/lustre/scratch/SOFTWARE/miniconda3/bin/chromosight \
detect \
--threads=8 \
--min-dist 20000 --max-dist 200000 \
/mnt/lustre/scratch/results.test/matrix/test.matrix.cool \
loops_H2087_intra
```

Thanks so much
Jorge

**Comment by cmdoret** [comment]
Hi Jorge,

Are you sure only this matrix was processed ? This commands works as expected on my machine.
Perhaps the matrix has very low coverage, try running again with `-z 100 -u 100` to scan all low coverage regions.

Also, does your cool file have contacts for all chromosomes ? what does the matrix look like if you do e.g.:

 `cooler show test.matrix.cool 'chr1:1000000-10000000'`

**Comment by jazberna1** [comment]
Hello Cyril,

My bad. All the matrices were processed it's just that the progress bar confused me as it finished in the EBV contig.

Then I run the _coller show_ command and got and empty plot which explains _the lack of patterns_ message.

I have now lowered the matrix resolution till I saw signal in the plot and set chromosight to run with -z 100 -u 100. Hopefully I'll get results now.

Again, thanks so much for being so helpful
Jorge
